packages/perseus/src/widgets/categorizer/categorizer.util.ts

```diff
@@ -0,0 +1,25 @@
+import type {PerseusCategorizerWidgetOptions} from "@khanacademy/perseus-core";
+
+type CategorizerSplitWidgetOptions = {
+    // a list of items to categorize. e.g. ["banana", "yellow", "apple", "purple", "shirt"]
+    items: PerseusCategorizerWidgetOptions["items"];
+    // a list of categories. e.g. ["fruits", "colors", "clothing"]
+    categories: PerseusCategorizerWidgetOptions["categories"];
+    // Whether the items should be randomized
+    randomizeItems: PerseusCategorizerWidgetOptions["randomizeItems"];
+    // Whether this widget is displayed with the results and immutable
+    static: PerseusCategorizerWidgetOptions["static"];
+};
```

Comment:
I'm of two minds on documenting these option fields. On the one hand, it might be useful for folks coming to this type to see what it does, but I think the risk of this comment getting out of sync with the main field in `PerseusCategorizerWidgetOptions` is greater. 

So I think I'd recommend removing all field documentation in this type and maybe adding a JSDoc string to the entire structure pointing back to the full options type for full details. 

Lastly, I think the `Split` term should be `Public` here also (so `CategorizerPublicWidgetOptions`). 

```suggestion
/** 
 * The `categorizer` widget's options that are public (ie. safe to 
 * pass to the browser without divulging the answer).
 * 
 * @see {@link PerseusCategorizerWidgetOptions} for full documentation of each option.
 */ 
type CategorizerPublicWidgetOptions = {
    items: PerseusCategorizerWidgetOptions["items"];
    categories: PerseusCategorizerWidgetOptions["categories"];
    randomizeItems: PerseusCategorizerWidgetOptions["randomizeItems"];
    static: PerseusCategorizerWidgetOptions["static"];
};
```